sass/components/_icons.scss

```diff
@@ -154,15 +160,18 @@
 
 .smartcharts {
     // temp workaround for svg icon that cannot be themed
-    .ic-cryptocurrency > svg, .ic-indices > svg {
+    .ic-cryptocurrency > svg,
+    .ic-indices > svg {
         stroke: none !important;
     }
     &-dark {
-        .ic-cryptocurrency > svg {
+        .ic-cryptocurrency > svg,
+        .ic-indices > svg {
```

Comment:
Lines 168-169 and 173-174 are the only changes that I have done. The rest is my prettier, cause this file fall off from project style agreements